src/main/java/util/FileHelper.java

```diff
@@ -0,0 +1,109 @@
+package util;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/**
+ * A helper that manages file I/O.
+ */
+public class FileHelper {
+    private static final Logger logger = LogManager.getLogger(FileHelper.class.getName());
+
+    private static final String CHARSET = "UTF-8";
+    private static final String EMPTY_STRING = "";
+
+    private final String configFileDirectory;
+    private final String configFileName;
+
+    /**
+     * Initialise the FileHelper with the directory and the file name
+     * @param configFileDirectory the directory that the config file is located in
+     * @param configFileName the name of the config file
+     */
+    public FileHelper(String configFileDirectory, String configFileName) {
+        this.configFileName = configFileName;
+        this.configFileDirectory = configFileDirectory;
+    }
+
+    /**
+     * Returns the File representation at the config file path
+     */
+    private File getFile() {
+        createDirectoryIfNotExist();
+        return new File(configFileDirectory, configFileName).getAbsoluteFile();
+    }
+
+    /**
+     * Attempts to create the config directory if it does not exist.
+     */
+    private void createDirectoryIfNotExist() {
+        File directory = new File(configFileDirectory);
+        boolean directoryExists = directory.exists() && directory.isDirectory();
+        if (!directoryExists) {
+            if (directory.mkdirs()) {
+                logger.info("Config directory created: " + directory.toString());
+            } else {
+                logger.warn("Could not create config file directory");
+            }
+        }
+    }
+
+    /**
+     * Load the contents of the file into a String
+     * @param ignoreExisting True if the existing file should be ignored
+     * @return The String representation of the contents of the config file
+     */
+    public String loadFileContents(boolean ignoreExisting) {
+        File configFile = getFile();
+        logger.info("Load starting: " + configFile.getAbsolutePath());
+
+        if (ignoreExisting) {
+            // ignore existing file, assume the file is new/empty, return empty string
+            logger.info("Ignoring existing file: " + configFile.getAbsolutePath());
+            return EMPTY_STRING;
+        }
+
+        if (!configFile.exists()) {
+            // no file, assume empty file, return empty string
+            logger.warn("Config file does not exist! Assuming empty file: " + configFile.getAbsolutePath());
+            return EMPTY_STRING;
```

Comment:
How about the IOExceptions in the file reading below? If we throw that, then the IO exception will overshadow the fileNotFound error. or.. should I not do the isExist check, and force the caller to handle the general IO exception instead?